test/e2e/storage/volume_provisioning.go

```diff
@@ -834,8 +833,7 @@ var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
 			}
 
 			ginkgo.By("creating a StorageClass")
-			suffix := fmt.Sprintf("invalid-aws")
-			class := newStorageClass(test, ns, suffix)
+			class := newStorageClass(test, ns, "invalid-aws")
 			class, err := c.StorageV1().StorageClasses().Create(context.TODO(), class, metav1.CreateOptions{})
```

Comment:
why this one does not use SetupStorageClass?

Reply:
When I did the refactor of `SetupStorageClass` I just looked at the places where the storage class was setup in `TestDynamicProvisioning()` and this method wasn't impacted by that refactor, I'll change this place too, thanks